components/infobox/extensions/commons/player_introduction.lua

```diff
@@ -396,7 +396,7 @@ function PlayerIntroduction:create()
 		return ''
 	end
 
-	local isDeceased = self.playerInfo.deathDate ~= '1970-01-01' or self.playerInfo.status == 'passed away'
+	local isDeceased = self.playerInfo.deathDate ~= '1970-01-01' or self.playerInfo.status == 'deceased'
```

Comment:
```suggestion
	local isDeceased = self.playerInfo.deathDate ~= '1970-01-01' or self.playerInfo.status == 'Deceased'
```
I believe

Reply:
no it gets lower cased in line 210:
`                status = Logic.emptyOr(args.status, playerInfo.status, 'active'):lower(),`

Reply:
Well that would be a breaking change, as most wikis would store it as `Active` atm 🤔 I think moving to `active` would make sense, but we would need to look at consumers.

Reply:
this is inside a consumer (player intro)
the person infobox stores upper first

Reply:
The current behavior on majority of the wiki is to store with an upper case first letter. So this PR then changes the behavior to always lowercase. While I think this is an improvement, there are consumers that are assuming it's `Active` and not `active` in LPDB

Reply:
Ah

Reply:
Ah, but that can probably be removed with these changes, but OOS I suppose

Reply:
it would need changes in several other lines in that module
i only changed the one string here as it was bs anyways